charts/datadog/values.yaml

```
@@ -1310,6 +1310,11 @@ clusterAgent:
   #          username: datadog
   #          password: <YOUR_CHOSEN_PASSWORD>
 
+  # clusterAgent.kubernetesApiserverCheck -- correspond to options for configuring the kube_apiserver integration.
```

```suggestion
  ## clusterAgent.kubernetesApiserverCheck -- correspond to options for configuring the kube_apiserver integration.
```
To not show up in the readme